.github/workflows/testing.yml

```diff
@@ -85,11 +82,8 @@ jobs:
           export KEEPER_RPC_PORT="18545"
           export KEEPER_NETWORK_NAME="spree"
           export KEEPER_OWNER_ROLE_ADDRESS="0xe2DD09d719Da89e5a3D0F2549c7E24566e947260"
-          export KEEPER_VERSION=latest
-          export GATEWAY_VERSION=develop
-          export MARKETPLACE_API_VERSION=latest
           export GANACHE_COMMAND_OPTIONS="-d -p 18545 -m 'taxi music thumb unique chat sand crew more leg another off lamp' -u 0xAFD49D613467c0DaBf47B8f5C841089d96Cf7167 -f https://eth-kovan.alchemyapi.io/v2/UaH4LeYR7hHlx00yQZanF1B5AUSmDsTA@28000000"
-          ./start_nevermined.sh --marketplace-api --no-marketplace --no-faucet --no-graph --local-ganache-node 2>&1 | tee nevermined_tools.txt &
+          ./start_nevermined.sh --latest --no-faucet --no-graph --local-ganache-node 2>&1 | tee nevermined_tools.txt &
```

Comment:
here the same question

Reply:
by the way the branch `set-rpc-port` was updated like the master in tools otherwise it will use `metada` instead of `marketplace-api`

Reply:
good question. I don't know what that branch does

Reply:
It looks like no https://github.com/nevermined-io/tools/tree/set-rpc-port then is better revert this part

Reply:
https://github.com/nevermined-io/tools/pull/110